draft-ietf-quic-transport.md

```diff
@@ -2178,12 +2180,16 @@ before the handshake is confirmed, as defined in section 4.1.2 of {{QUIC-TLS}}.
 
 If the peer sent the disable_active_migration transport parameter, an endpoint
 also MUST NOT send packets (including probing packets; see {{probing}}) from a
-different local address to the address the peer used during the handshake. An
-endpoint that has sent this transport parameter, but detects that a peer has
-nonetheless migrated to a different remote address MUST either drop the incoming
-packets on that path without generating a stateless reset or proceed with path
-validation and allow the peer to migrate. Generating a stateless reset or
-closing the connection would allow third parties in the network to cause
+different local address to the address the peer used during the handshake. If
+the server sets the Disable Migration field in the preferred_address transport
+parameter, the client MUST NOT send packets from a different local address to
```

Comment:
I think we could replace MUST with SHOULD in regard to `disable_active_migration` and `Disable Migration`.  Nothing here is enforceable, since unintended migrations can always happen.  The effect is likely similar.

Reply:
That's a totally different discussion that we've had a few times already. Happy to have it again, but let's do that in the issue, or a different issue, but certainly not in this PR.